src/components/VIcon/VIcon.vue

```diff
@@ -51,11 +51,15 @@ export default {
     },
     /**
      * the size of the icon in terms of the Tailwind height and width scale; The
-     * icon, being square, uses this for both dimensions.
+     * icon, being square, uses this for both dimensions. Possible values:
+     * 4 - 16px(1rem), 5 - 20px(1.25rem), 6 - 24px(1.5rem).
```

Comment:
Let's not reference `px` values here. Keeping the numeric sizes decoupled from actual values seems better to me.

Reply:
Probably best not to mention specific values in the comment at all—typedoc or a link to the tailwind config would be ideal.

Reply:
We should still clarify that the acceptable values are 4, 5 and 6 only.